tests/unit/dogstatsd/test_statsd.py

```diff
@@ -313,7 +313,9 @@ def test_timed_context(self):
         Measure the distribution of a context's run time.
         """
         # In seconds
-        with self.statsd.timed('timed_context.test'):
+        with self.statsd.timed('timed_context.test') as timer:
+            self.assertIsInstance(timer,
```

Comment:
Interesting. I've haven't run across `nose.tools` being used before. Should be in place now.